### DISCUSSION TITLE
Het runnen van bagv2/etl/etl.sh, zet opnieuw het schema neer maar mist dan postgis

### DISCUSSION THREAD

**Discussion by hawkrevenge** [nlextract/NLExtract#370] [Q&A] [answered]
Bij mijn psql database (v13.2) op OSX 11.6 (geen idee of dat hierbij uit maakt) wordt door het schema opnieuw in te laden de postgis extentie verwijderd. Aangezien hij deze nergens opnieuw in laadt lijkt het mij een goede deze regel toe te voegen aan het bagv2/etl/sql/create-schema.sql:

CREATE EXTENSION postgis SCHEMA {schema};

Zonder deze extentie loopt de het aanmaken van de tabellen stuk op het type geometry

**Comment by justb4** [comment, marked as answer]
Vreemd, kan het zijn dat je `public` schema gebruikt? Daarin zit PostGIS data.
